db/migrate/20221013045158_add_sidebar_list_destination_to_user_option.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSidebarListDestinationToUserOption < ActiveRecord::Migration[7.0]
+  def change
+    add_column :user_options, :sidebar_list_destination, :integer
```

Comment:
I see that we removed the default here but I think an enum requires a default?

Reply:
I recall there is a hack that allows admins to set default, null means a user did not choose yet?

Reply:
Looks like we need to use the `_default` option when declaring the enum on the model.

Reply:
Ah, you are right, default is added as value from SiteSetting:

  enum sidebar_list_destination: { latest: 0, unread_new: 1 }, _scopes: false, _default: SiteSetting.default_sidebar_list_destination